qiskit/circuit/quantumcircuit.py

```diff
@@ -3413,13 +3481,7 @@ def num_nonlocal_gates(self) -> int:
 
         Conditional nonlocal gates are also included.
         """
-        multi_qubit_gates = 0
-        for instruction in self._data:
-            if instruction.operation.num_qubits > 1 and not getattr(
-                instruction.operation, "_directive", False
-            ):
-                multi_qubit_gates += 1
-        return multi_qubit_gates
+        return self._data.num_nonlocal_gates()
```

Comment:
Did this _need_ to change?  I'm a bit worried if it did.

Reply:
Not at all, I did this to see a performance win (its a 27x speed up). I really should have saved it for a follow up pr, but it was small enough it didn't feel egregious.